main.nf

```diff
@@ -293,7 +293,15 @@ publish_pattern_salmon_ga = params.salmon_keep_data
  */
 DONE_SENTINEL = 1
 
-
+/**
+ * Clean up any files left over from a previous run by moving them
+ * back to the stage directory.
+ */
+existing_files = file('work/GEMmaker/process/*')
+for (existing_file in existing_files) {
+  existing_file.moveTo('work/GEMmaker/stage')
+}
+existing_files = null
```

Comment:
This part is already done at the beginning of the workflow block (line 309-319). So either this code should be deleted or the existing code should be moved up if that's what you want. The other code is more robust in my opinion.

Reply:
Oh, I didn't notice that code. You're right.  I'll remove the lines I added.

Reply:
Oh, I didn't notice that code. You're right.  I'll remove the lines I added.